sample_project/Assets/SampleViewer/Samples/FeatureLayer/Scripts/ArcGISFeatureLayerComponent.cs

```diff
@@ -0,0 +1,142 @@
+// Copyright 2025 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+using Esri.GameEngine.Geometry;
+using Esri.ArcGISMapsSDK.Utils.GeoCoord;
+using Esri.ArcGISMapsSDK.Components;
+using System;
+
+public class ArcGISFeatureLayerComponent : MonoBehaviour
+{
+    [System.Serializable]
+    public struct QueryLink
+    {
+        public string Link;
+        public string[] RequestHeaders;
+    }
+
+    [System.Serializable]
+    public class FeatureQueryData
+    {
+        public GeometryData geometry = new GeometryData();
+        public PropertyData properties = new PropertyData();
+    }
+
+    [System.Serializable]
+    public class PropertyData
+    {
+        public List<string> propertyNames = new List<string>();
+        public List<string> data = new List<string>();
+    }
+
+    [System.Serializable]
+    public class GeometryData
+    {
+        public List<double> coordinates = new List<double>();
+    }
+
+    public List<FeatureQueryData> Features = new List<FeatureQueryData>();
+    private FeatureData featureInfo;
+    public List<GameObject> FeatureItems = new List<GameObject>();
+    [SerializeField] private GameObject featurePrefab;
+    private float spawnHeight = 10000.0f;
+
+    public JToken[] jFeatures;
+    public QueryLink webLink;
+
+    public void CreateLink(string link)
+    {
+        if (link != null)
+        {
+            foreach (var header in webLink.RequestHeaders)
+            {
+                if (!link.ToLower().Contains(header))
+                {
+                    link += header;
+                }
+            }
+
+            webLink.Link = link;
+        }
+    }
+
+    public IEnumerator GetFeatures()
+    {
+        // To learn more about the Feature Layer rest API and all the things that are possible checkout
+        // https://developers.arcgis.com/rest/services-reference/enterprise/query-feature-service-layer-.htm
+
+        UnityWebRequest Request = UnityWebRequest.Get(webLink.Link);
+        yield return Request.SendWebRequest();
+
+        if (Request.result != UnityWebRequest.Result.Success)
+        {
+            Debug.Log(Request.error);
+        }
+        else
+        {
+            CreateGameObjectsFromResponse(Request.downloadHandler.text);
+        }
+    }
+
+    private void CreateGameObjectsFromResponse(string response)
+    {
+        // Deserialize the JSON response from the query.
+        var jObject = JObject.Parse(response);
```

Comment:
jResponse?

Reply:
The response is just a string. The JObject variables are Json Objects